dashboard/app/jobs/concerns/active_job_metrics.rb

```diff
@@ -148,9 +148,26 @@ def self.report_metrics(job_class, dimensions:)
   end
 
   def self.report_overall_queue_metrics
+    migrate_failed_jobs
```

Comment:
I _really_ don't love a migration happening as a surprise side effect of reporting metrics. Can we split this off into its own thing?

Reply:
Yeah, I can make a separate cron job, likely running less frequently, to handle the cleanup migration.